modules/dnn/src/dnn.cpp

```diff
@@ -283,7 +283,7 @@ void blobFromImages(InputArrayOfArrays images_, OutputArray blob_, double scalef
         images[i] *= scalefactor;
     }
 
-    size_t i, nimages = images.size();
+    size_t nimages = images.size();
```

Comment:
There is declaration of this variable below (line 290):

> Mat image;

Its scope can be reduced too.